### PR TITLE
Silent failures are disabled by default

### PR DESCRIPTION
**Pull request by pmitros**
This is the mate to https://github.com/edx/XBlock/pull/284

DO NOT MERGE: Before this is merged, the XBlock PR needs to be accepted, and requirements.txt must be updated to point to the XBlock.

@doctoryes @cpennington

### CODE REVIEW

File: workbench/scenarios.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def remove_scenario(scname):
     del SCENARIOS[scname]
 
 
-def add_class_scenarios(class_name, cls):
+def add_class_scenarios(class_name, cls, fail_silently = True):
```
Comment (doctoryes) [inline workbench/scenarios.py]: Nit: No whitespace around '='

File: workbench/scenarios.py [hunk 2]
```diff
@@ -69,5 +71,5 @@ def init_scenarios():
         WORKBENCH_KVS.prep_for_scenario_loading()
 
     # Get all the XBlock classes, and add their scenarios.
-    for class_name, cls in sorted(XBlock.load_classes()):
-        add_class_scenarios(class_name, cls)
+    for class_name, cls in sorted(XBlock.load_classes( fail_silently = False )):
+        add_class_scenarios(class_name, cls, fail_silently = False)
```
Comment (doctoryes) [inline workbench/scenarios.py]: Nit: No whitespace around '='

**Comment by doctoryes** [comment]
Looks like all failing tests are due to the old XBlock version? 

After all tests pass, :+1:

**Comment by pmitros** [comment]
@cpennington ping?

**Comment by cpennington** [comment]
:+1:

**Pull request merged**